aep/general/0300/aep.md.j2

```diff
@@ -0,0 +1,98 @@
+# AEP Editions
+
+This AEP describes how the AEP specification is versioned.
+
+## Edition based versioning
+
+AEPs will be published in editions, similar but not identical to specifications
+such as those for the C++ or Rust programming language:
+
+- The specification will not change for a non-preview edition after it is
+  published.
+- The naming of the edition will be based on the year in which it is published
+  (e.g. aep-2025).
+- Upon publishing a non-preview edition, the current prevision will be removed,
```

Comment:
I'd remove `the current prevision will be removed,`. We haven't used the terminology 'prevision' before and I think it muddles the message.

Reply:
```suggestion
- Upon publishing a non-preview edition, the current preview edition will be removed,
```

I think we should write out "preview edition" (I'm assuming prevision was either a typo or an intended contraction of "preview edition").

Reply:
sorry, typo on my end.